src/api/subscriptions.js

```diff
@@ -279,6 +285,16 @@ export const ReplaceSkuProrationMode = {
   IMMEDIATE_WITH_TIME_PRORATION: 'IMMEDIATE_WITH_TIME_PRORATION',
 };
 
+/**
+ * The product type. The Offers UI is rendered differently based on the
+ * ProductType and 'product_type' parameter is passed to Payments flow to render
+ * the GPay content accordingly. ProductType Defaults to 'SUBSCRIPTION'.
+ * @enum {string}
+ */
+export const ProductType = {
+  SUBSCRIPTION: 'SUBSCRIPTION',
+  UI_CONTRIBUTION: 'UI_CONTRIBUTION',
```

Comment:
nit: why is this UI_CONTRIBUTION? Why not just CONTRIBUTION?

Reply:
This type is defined by Play. The main reason is that the impact is only in the UI, the internal details remain same.
The enum is defined in the Play.